screens/auth/LogInScreen.js

```diff
@@ -44,6 +45,7 @@ export default class LogInScreen extends React.Component {
   // From SignUpScreen. Sign in function. It sets the user token in local storage
   // to be the user ID and then navigates to homescreen.
   _asyncLogIn = async customerId => {
+    await Analytics.setUserId(customerId);
```

Comment:
suggestion is to move this right above where Sentry registers a use on successful login  on line 114
```
Sentry.configureScope(scope => {
```